src/components/Main/Main.css

```diff
@@ -0,0 +1,18 @@
+.country {
+  display: flex;
+  align-items: center;
+  align-content: flex-start;
+  flex-direction: column;
+  flex-wrap: wrap;
+  height: 100vh;
+  width: 100vw;
+  margin-top: 25px;
+}
+
+.select {
+  display: flex;
+  position: absolute;
```

Comment:
this absolute positioning makes the ```<select>``` straddle the header/main on my computer, not sure if that's what you were going for